src/Core/Metadata/Delete.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Metadata;
+
+#[\Attribute(\Attribute::TARGET_CLASS | \Attribute::IS_REPEATABLE)]
+class Delete extends Operation
```

Comment:
```suggestion
final class Delete extends Operation
```

Reply:
Same for other operations

Reply:
IMO we should let this non-final so that people can extend our operations